### ISSUE TITLE
Change button 'Edit' and 'More information' to icons

### ISSUE THREAD

**Issue by kamgrzeski** [kamgrzeski/SoftCRM#5]

**Comment by WesleyNobrega** [comment]
Dear Kamil,

How are things going in times of pandemic? It's crazy here, we're all home, if you go out. I took the opportunity to take a look at the codes, I came across the data entries as follows:
 <div class = "form-group input-row">
   {{Form :: label ('zip', 'ZIP')}}
   <div class = "input-group">
      <span class = "input-group-addon"> <i class = "fa fa-registered"> </i> </span>
          {{Form :: text ('zip', null, ['class' => 'form-control', 'placeholder' => App \ Traits \ Language :: 
         getMessage ('messages.InputText')])}}
    </div>
</div>
This type of input does not provide an option for a data entry mask, a good example is ZIPCODE, which here has the format 99.999-999. How could we do to modify these INPUT by placing a fixed mask?
